From 2ecea38dc2c6014b307493e4731f1640656d2bcf Mon Sep 17 00:00:00 2001
From: zhouming <B42586@freescale.com>
Date: Thu, 11 Oct 2012 14:46:10 +0800
Subject: [PATCH 123/602] ENGR00133115: add amr parser for pure amr audio

add lib_amr_parser_arm11_elinux.3.0 in the omx makefile to add amr parser

Signed-off-by: zhouming <B42586@freescale.com>
---
 imx5x/imx5x.mk | 3 ++-
 imx6/imx6.mk   | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/imx5x/imx5x.mk b/imx5x/imx5x.mk
index fde82fb..3cce5d1 100644
--- a/imx5x/imx5x.mk
+++ b/imx5x/imx5x.mk
@@ -221,7 +221,8 @@ omx_libs := \
 	libfsl_jpeg_enc_arm11_elinux			\
 	lib_wb_amr_enc_arm11_elinux			\
 	lib_wb_amr_dec_arm9_elinux			\
-	lib_omx_aac_enc_v2_arm11_elinux
+	lib_omx_aac_enc_v2_arm11_elinux \
+	lib_amr_parser_arm11_elinux.3.0 \
 
 # Omx excluded libs
 omx_excluded_libs :=					\
diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 830c66e..76bb9e2 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -222,7 +222,8 @@ omx_libs := \
 	libfsl_jpeg_enc_arm11_elinux \
 	lib_wb_amr_enc_arm11_elinux \
 	lib_wb_amr_dec_arm9_elinux \
-	lib_omx_aac_enc_v2_arm11_elinux
+	lib_omx_aac_enc_v2_arm11_elinux \
+	lib_amr_parser_arm11_elinux.3.0 \
 
 # Omx excluded libs
 omx_excluded_libs :=					\
-- 
1.8.0

